docker/docker-compose.2004.main.insiders.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+
+  runtime-setup:
+    image: vscode-swift:22.04-main
```

Comment:
do we maybe do 6.0, i.e. latest release so we know the toolchain is at least more or less stable? Nightly toolchain and nightly VS Code could make it a little harder to know the failure source

Reply:
Good point, probably better not to have two moving targets.